plugins/guests/linux/cap/public_key.rb

```diff
@@ -53,6 +53,7 @@ def self.remove_public_key(machine, contents)
               result=$?
             fi
             rm -f '#{remote_path}'
+            chmod 0600 ~/.ssh/authorized_keys
```

Comment:
Hey there! Thanks for your contribution. I have a question....

It looks like this step is already happening and chmod'ing authorized_keys in the `if` block above. Do you have a reproduction case where it's failing, requiring this change? Thank you!